pkg/cli/admin/release/extract_tools.go

```diff
@@ -632,6 +644,51 @@ const (
 	releaseVersionMarker = "!_RELEASE_VERSION_LOCATION_\x00XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\x00"
 )
 
+func getValidTarget(spec, dir, releaseName string, target extractTarget, willArchive bool) (extractTarget, bool, error) {
+	ref, err := imagereference.Parse(spec)
```

Comment:
This won't be necessary if you pass `DockerImageReference` as mentioned before.

Reply:
that will give the release sha but not the release-tool

Reply:
resolving this, since i've moved all changes to pkg/cli/admin/info